### DISCUSSION TITLE
ModuleNotFoundError when intalling dpgen

### DISCUSSION THREAD

**Discussion by hu-lm** [deepmodeling/dpgen#1118] [Q&A]
Hi,

I was trying to install dpgen and run it on my HPC workstation. However, the following error keeps occurring, I have been attempting both download the source code to install and directly pip install, and both of them have this issue. Also, the package pymatgen is already installed (I have checked #963, and seems like it should work even if my pymatgen is the 2023 version?). I would appreciate it if someone could help me figure this out. Thank you!

<img width="744" alt="image" src="https://user-images.githubusercontent.com/98113070/212757745-ef2e0958-b205-4346-9106-f19aee01dc3b.png">

Best,

**Comment by njzjz** [comment]
It looks that the code you ran was not the latest. You may check your code to see if #963 is patched correctly.
